### PR TITLE
Rename SYSCALLs

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/neo/SmartContract/InteropService.0.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+using Neo.VM;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.SmartContract
+{
+    public static partial class InteropService
```
Comment (shargon) [inline src/neo/SmartContract/InteropService.0.cs]: I prefer only `interopservice` instead of `Interopservice.0` as filename

Comment (erikzhang) [reply src/neo/SmartContract/InteropService.0.cs]: I add 0 to ensure the order of the initialization of the static fields.

**Comment by erikzhang** [comment]
Only rename and split to multiple files.

File: src/neo/SmartContract/InteropService.0.cs (hunk 1 above)
Comment (erikzhang) [reply src/neo/SmartContract/InteropService.0.cs]: Improved.

**Pull request merged**